cea/optimization/master/evaluation.py

```diff
@@ -306,15 +312,15 @@ def calc_master_to_slave_variables(individual, Qmax, locator, gv, optimization_c
 
     shareAvail = areaAvail / totalArea    
     
-    irank = optimization_constants.nHeat * 2 + optimization_constants.nHR
+    irank = nHeat * 2 + nHR
     master_to_slave_vars.SOLAR_PART_PV = max(individual[irank] * individual[irank + 1] * individual[irank + 6] * shareAvail,0)
     master_to_slave_vars.SOLAR_PART_PVT = max(individual[irank + 2] * individual[irank + 3] * individual[irank + 6] * shareAvail,0)
     master_to_slave_vars.SOLAR_PART_SC = max(individual[irank + 4] * individual[irank + 5] * individual[irank + 6] * shareAvail,0)
 
     return master_to_slave_vars
 
 
-def checkNtw(individual, ntwList, locator, gv, optimization_constants):
+def checkNtw(individual, ntwList, locator, gv):
```

Comment:
and when we do get around to some refactoring, this could be called `check_network` - abreviations should be used sparingly...